src/runtime-rs/crates/resource/src/network/network_with_netns.rs

```diff
@@ -149,7 +152,10 @@ impl Network for NetworkWithNetns {
     }
 }
 
-async fn get_entity_from_netns(config: &NetworkWithNetNsConfig) -> Result<Vec<NetworkEntity>> {
+async fn get_entity_from_netns(
+    config: &NetworkWithNetNsConfig,
+    d: Arc<RwLock<DeviceManager>>,
+) -> Result<Vec<NetworkEntity>> {
     info!(
         sl!(),
         "get network entity for config {:?} tid {:?}",
```

Comment:
Is it "for" or "from" in the log `get network entity for config` ?

Reply:
yes, fixed